mail/tasks_test.py

```diff
@@ -10,20 +10,32 @@
 from mail import tasks
 from mail.models import NotificationEmail
 from ui.constants import VideoStatus
-from ui.factories import VideoFactory
+from ui.factories import VideoFactory, CollectionFactory, MoiraListFactory
 
 pytestmark = pytest.mark.django_db
 
 
 # pylint: disable=protected-access
 
 
-def test_get_recipients_for_video():
+@pytest.fixture(autouse=True)
+def mocker_defaults(mocker):
+    """
+    Sets default settings to safe defaults
+    """
+    mocker.patch('mail.tasks.has_common_lists', return_value=False)
+
+
+def test_get_recipients_for_video(mocker):
     """
     Tests the _get_recipients_for_video api
     """
-    video = VideoFactory()
-    assert tasks._get_recipients_for_video(video) == [video.collection.owner.email]
+    lists = [MoiraListFactory(), MoiraListFactory()]
+    video = VideoFactory(collection=CollectionFactory(admin_lists=lists))
```

Comment:
This _might_ work...

`video = VideoFactory(collection__admin_lists=lists)`

Reply:
Works!